src/consent/types.ts

```diff
@@ -87,3 +93,9 @@ export type ConsentProps =
   | ConsentPropsTCFV2
   | ConsentPropsNonTCFV1
   | ConsentPropsUnified;
+
+export interface MSPATransactions {
```

Comment:
Think this should probably be changed for consistency with its properties.
```suggestion
export interface MspaTransactions {
```

Reply:
Good catch, but this type is no longer needed.